fe/fe-core/src/main/java/com/starrocks/sql/ast/KillStmt.java

```diff
@@ -24,17 +24,25 @@
  * KILL [QUERY | CONNECTION] connection_id
  */
 public class KillStmt extends StatementBase {
+    private final boolean isGlobal;
     private final boolean isConnectionKill;
     private final long connectionId;
+    private final String queryId;
 
-    public KillStmt(boolean isConnectionKill, long connectionId) {
-        this(isConnectionKill, connectionId, NodePosition.ZERO);
+    public KillStmt(boolean isGlobal, boolean isConnectionKill, long connectionId, String queryId) {
```

Comment:
We'd better create three separate constructors here.
1. for KILL query by query id
2. for KILL query by connection id
3. for KILL connection by connection id

Reply:
Modified